src/BenchmarkDotNet.Core/Exporters/Xml/XmlExporterBase.cs

```diff
@@ -10,38 +12,41 @@ public abstract class XmlExporterBase : ExporterBase
     {
         protected override string FileExtension => "xml";
 
-        private readonly XmlWriterSettings settings;
+        private readonly bool indentXml;
         private readonly bool excludeMeasurements;
 
         public XmlExporterBase(bool indentXml = false, bool excludeMeasurements = false)
         {
-            settings = new XmlWriterSettings
-            {
-                Indent = indentXml
-            };
-
+            this.indentXml = indentXml;
             this.excludeMeasurements = excludeMeasurements;
         }
 
         public override void ExportToLog(Summary summary, ILogger logger)
         {
-            var serializer = new XmlSerializer(typeof(SummaryDto))
-                                    .WithRootName(nameof(Summary))
-                                    .WithCollectionItemName(typeof(Measurement),
-                                                            nameof(Measurement))
-                                    .WithCollectionItemName(typeof(BenchmarkReportDto),
-                                                            nameof(BenchmarkReport.Benchmark));
+            IXmlSerializer serializer =
+                new XmlSerializer(typeof(SummaryDto))
+                    .WithRootName(nameof(Summary))
```

Comment:
I really like this fluent api! 👍 

Btw the next step to make it even better would be to create a fluent `XmlSerializerBuilder` with mutable private fields accessed via your public fluent api. Then at the end you could call `builder.Build()` and receive immutable serializer with `Serialize` method only.